base.requirements.txt

```diff
@@ -1,4 +1,5 @@
 ## Base requirements
+dojson==0.4.0
```

Comment:
shall we add here https://github.com/inspirehep/inspire-next/blob/master/setup.py#L32 instead?